WordPressMocks/src/main/assets/mocks/mappings/wpcom/activity/wpcom_v2_site_activity.json

```diff
@@ -38,7 +38,7 @@
                             "name": "Jetpack"
                         },
                         "type": "Announce",
-                        "published": "2021-05-06T06:23:10.798+00:00",
+                        "published": "{{now format='yyyy-MM-dd'}}T{{now format='HH:mm:ss.SSSZ'}}",
```

Comment:
Unrelated to this PR but needs to be fixed in a separate PR:

We need to use `fnow` instead of `now` to avoid the issue with some locales described in pbMoDN-13F-p2

(We might also want to add some rudimentary lint – maybe just a simple `grep {{ *now` could do – to avoid this in the future)

Reply:
Good point 🤔 I replied in the private P2 because that's where all the context of the `fnow` conversation lives.